src/BizHawk.Emulation.Cores/Consoles/Nintendo/BSNES/BsnesCore.cs

```diff
@@ -10,11 +10,13 @@ namespace BizHawk.Emulation.Cores.Nintendo.BSNES
 {
 	[PortedCore(CoreNames.Bsnes115, "bsnes team", "v115+", "https://github.com/bsnes-emu/bsnes")]
 	[ServiceNotApplicable(new[] { typeof(IDriveLight) })]
-	public unsafe partial class BsnesCore : IEmulator, IDebuggable, IVideoProvider, ISaveRam, IStatable, IInputPollable, IRegionable, ISettable<BsnesCore.SnesSettings, BsnesCore.SnesSyncSettings>
+	public unsafe partial class BsnesCore : IEmulator, IDebuggable, IVideoProvider, ISaveRam, IStatable, IInputPollable,
+		IRegionable, ISettable<BsnesCore.SnesSettings, BsnesCore.SnesSyncSettings>, ICycleTiming
```

Comment:
I suggest not adding an ICycleTiming here, as otherwise the movie recorded will complain if cycle count is not present in movie (not needed as subframe is a different core).